bin/compile

```diff
@@ -9,8 +9,8 @@ for import in "$(dirname ${bin_dir})/lib/"*; do
   source "$import"
 done
 
-mkdir -p $build_dir/public/assets
-mkdir -p $build_dir/public/packs
+mkdir -p $build_dir/assets-$SOURCE_VERSION/public/assets
+mkdir -p $build_dir/assets-$SOURCE_VERSION/public/packs
```

Comment:
Maybe we should do like `/assets/$SOURCE_VERSION/`?  That way the top level won't get filled up with commit SHAs.